bin/crew

```diff
@@ -891,11 +891,11 @@ def prepare_package(destdir)
     # than install-info.
     # https://www.debian.org/doc/debian-policy/ch-docs.html#info-documents
     FileUtils.rm "#{CREW_DEST_PREFIX}/share/info/dir" if File.exist?("#{CREW_DEST_PREFIX}/share/info/dir")
-    
+
```

Comment:
Thank you for removing trailing whitespace.

Reply:
I think my IDE automatically did that, I definitely didn't.

Reply:
Thank you for configuring your IDE to remove trailing whitespace. :)